swaybar/ipc.c

```diff
@@ -7,6 +7,16 @@
 #include "bar/config.h"
 #include "bar/ipc.h"
 
+void ipc_send_workspace_command(const char *workspace_name) {
+	sway_log(L_DEBUG, "Clicked on workspace %s", workspace_name);
```

Comment:
This log thing is not necessarily true. Anything could have invoked this function, not just mouse click things.